app/controllers/metrics_controller.rb

```diff
@@ -7,13 +7,16 @@ def index
       @dataset = CalculatedMetrics.new(VisitMetricsEntry.deferred, 3.days)
     else
       @start_date, @end_date = fortnightly_range.first, fortnightly_range.last
-      @dataset = CalculatedMetrics.new(VisitMetricsEntry.deferred, 3.days, fortnightly_range)
+      @dataset = CalculatedMetrics.new(
+        VisitMetricsEntry.deferred, 3.days, fortnightly_range)
     end
 
     respond_to do |format|
       format.html
       format.csv do
-        render text: CSVFormatter.new(@nomis_ids, ApplicationHelper.instance_method(:prison_name_for_id)).generate(@dataset)
+        render text: CSVFormatter.new(
+          @nomis_ids, ApplicationHelper.instance_method(:prison_name_for_id)).
+        generate(@dataset)
```

Comment:
I think this is a good indication of where putting `)` on the next line helps. At a first read, it's not clear that `generate` follows on, whereas

```ruby
).generate(@dataset)
```

is more obvious.